src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -284,6 +286,72 @@ For example, the path for german would be ``src/locale/messages.de-DE.xlf``.
 ``<locale-id>`` should match the id previouisly inserted in
 ``supported-languages.enum.ts``.
 
+Suggestions
+~~~~~~~~~~~
+
+Strings need to start and end in the same line as the element,
```

Comment:
Nit: how about using colons instead of commas here?

Reply:
my bad, misread your previous comment...

Reply:
Done.